### PR TITLE
Make ThreatConnect pip package

### PR DESCRIPTION
**Pull request by jake223**
I have created a `setup.py` and so on for `pip` to be able to install `ThreatConnect` python package. Try installing with:

    pip install git+git://github.com/percipient/TCPy_V2.git@PyPI

I have chosen the MIT license because a license is required, but when I pull-request ThreatConnect I'll ask them which license they want me to use.

### CODE REVIEW

**Comment by clokep** [comment]
[This](https://github.com/ThreatConnect-Inc/TCPy_V2/blob/678d31efb32ecbdbffff2b8579130803909f6296/ThreatConnect/__init__.py#L18) seems to say GPLv3.

**Comment by jake223** [comment]
I have changed everything to Apache 2.0 license on the basis of an email from a ThreatConnect person.

**Comment by clokep** [comment]
We should mark `requests` as a requirement and install it automatically, (this is explained in https://hynek.me/articles/sharing-your-labor-of-love-pypi-quick-and-dirty/)

I tested this by making a new virtualenv and installing this.

```bash
mkdir temp && cd temp
mkvirtualenv temp
pip install https://codeload.github.com/percipient/TCPy_V2/zip/PyPI
```

I then ran Python and attempted to `import ThreatConnect`, which failed with trying to import `requests`.

Please modify the description with detailed steps to test that this installed properly. Thanks!

**Comment by jake223** [comment]
Requests is now a dependency. This can be tested as follows:

    pip install git+git://github.com/percipient/TCPy_V2.git@PyPI
    python 

In python:

    from ThreatConnect import ThreatConnect
    tc = ThreatConnect(your API parameters)
    print tc.get_groups.data().name_list() # should print a list of names of groups known to the org whose API key this is.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    url='https://github.com/ThreatConnect-Inc/TCPy_V2',
+    license='Apache 2.0',
+    author='Drew Gidwani',
+    author_emai='dgidwani@threatconnect.com',
+    packages=find_packages(),
+    install_requires=['requests'],
+    classifiers=[
+        "Natural Language :: English",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python :: 2.6",
```
Comment (clokep) [inline setup.py]: Did we actually check this on 2.6 (or do they say it's compatible)? If not, don't mark it as compatible.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        "Programming Language :: Python :: 2.7",
+        "Topic :: Security",
+        "Topic :: Internet",
+
+    ]
+
+)
```
Comment (clokep) [inline setup.py]: There's a few odd empty lines in here, please remove lines 32, 30, and 11.

**Comment by clokep** [comment]
A couple of minor nits to fix, and then we should squash this into a single commit and I will help you put up a pull request with upstream.

**Pull request closed without merging**